pkg/storage/endpoint/tso_keyspace_group.go

```diff
@@ -22,6 +22,44 @@ import (
 	"go.etcd.io/etcd/clientv3"
 )
 
+// UserKind represents the user kind.
+type UserKind int
+
+// Different user kinds.
+const (
```

Comment:
shall we define it as enum in keyspace.proto then we get all the following functions for free?

Reply:
Good question, we need a discussion about it.

Reply:
From my perspective, it's ok to keep it for now since we don't know if the current definition meets the future requirement. It's easier to modify compared with using proto.